### ISSUE TITLE
Permission denied

### ISSUE THREAD

**Issue by Mammutor** [phanirithvij/twist.moe#26]
I have an Issue where i dont get the list.txt
I think because of the "Permission denied"

Mammu@DESKTOP-RQPGT3I MINGW64 ~/twist.moe (master)
$ ./startp.sh gintama
URL: https://twist.moe/a/gintama
making it https://twist.moe/a/gintama/1
Fetching info...
rm: cannot remove '.temp.txt': No such file or directory
../api.sh: line 9: /c/Users/Mammu/AppData/Local/Microsoft/WindowsApps/python3: Permission denied
../api.sh: line 14: /c/Users/Mammu/AppData/Local/Microsoft/WindowsApps/python3: Permission denied
Creating gintama/list.txt
copying download.sh to gintama
Done
Check the Anime/gintama folder
Enjoy
Done

Mammu@DESKTOP-RQPGT3I MINGW64 ~/twist.moe/anime/gintama (master)
$ ./download.sh
anime is gintama
curl is installed: yes
cat: list.txt: No such file or directory
Total no of episodes : 0
Enter 'a' for all episodes
start at:
a
cat: list.txt: No such file or directory

**Comment by phanirithvij** [comment]
Hi have you installed python3? The python path that's in your error message
seems like it is the one shipped with windows. Please install the latest
version of python3 and add it to your PATH. Detailed instructions here.
https://docs.python.org/3/using/windows.html#windows-full

**Comment by Mammutor** [comment]
![FdRD1pl](https://user-images.githubusercontent.com/44206081/81818087-dbaf7f00-952d-11ea-9529-3d021735e3e2.png)
I have the latest version and I checked the mark and I see it there in PATH

**Comment by phanirithvij** [comment]
Ok, can you do a `where python` and let me know the results?
After that, you need to modify the api.sh file to use the newly installed
python instead of that `.../WindowsApps/python3` one.

**Comment by Mammutor** [comment]
What is `where python` and how i execute it? And I dont understand what I have to modify in the api.sh

**Comment by phanirithvij** [comment]
You can fix the issue in two ways

In cmd (search cmd in windows search) or cmder (if you have installed it) run the command `where python` and also run `where python3` these commands will show all the python installations in your system. Then in the `twist.moe` directory that you cloned you can see `api.sh` when you do an `ls -l` in cmd or cmder. Open the file using notepad or some text editor and where there is python3 replace it with the newly installed python3's path.

## OR

In your PATH you have `C:\Users\Mammu\AppData\Local\Microsoft\WondowsApps` if it's not necessary you can remove it. Or if you need it in PATH then change the order in the path i.e. first you'll have `C:\Users\Mammu\AppData\Local\Microsoft\WondowsApps` after that you need to have `C:\Users\Mammu\AppData\Local\Programs\Python\Python38`

Finally it should be like
Path is
```
C:\Users\Mammu\AppData\Local\Programs\Python\Python38\Scripts;C:\Users\Mammu\AppData\Local\Microsoft\WondowsApps;C:\Users\Mammu\AppData\Local\Programs\Python\Python38
```

**Comment by Mammutor** [comment]
I have removed the `C:\Users\Mammu\AppData\Local\Microsoft\WondowsApps`. The first way does not work:
 
Mammu@DESKTOP-RQPGT3I ~/twist.moe (master)
λ where python
bash: where: command not found

So I hope that I got I now right.
![N5AC09m](https://user-images.githubusercontent.com/44206081/82283722-79280a00-9997-11ea-96e1-98a54e81535e.png)

But I have now a different problem. 

Mammu@DESKTOP-RQPGT3I ~/twist.moe (master)
λ ./startp.sh gintama
URL: https://twist.moe/a/gintama
making it https://twist.moe/a/gintama/1
Fetching info...
rm: cannot remove '.temp.txt': No such file or directory
../api.sh: line 9: python3: command not found
../api.sh: line 14: python3: command not found
Creating gintama/list.txt
copying download.sh to gintama
Done
Check the Anime/gintama folder
Enjoy
Done

**Comment by phanirithvij** [comment]
Ok open the api.sh file in the twist.moe directory using notepad and change python3 to `py` everywhere

**Comment by Mammutor** [comment]
same issue
![c4rNo3B](https://user-images.githubusercontent.com/44206081/82619302-de693e80-9bd5-11ea-9ca3-ca7c05ff164b.png)
I also tried the setup command and i get this
![P79Yv0r](https://user-images.githubusercontent.com/44206081/82620368-9a2b6d80-9bd8-11ea-8b14-34a235181b28.png)
Just looked up the error. I saw that command and I remember that you have said something about an "where python" command. I hope that can help 
![kJViEgk](https://user-images.githubusercontent.com/44206081/82620815-bb408e00-9bd9-11ea-87d9-e37a3d714479.png)

**Comment by phanirithvij** [comment]
Ok go to that C:\Users\..Python\Python38 and do an `ls` see if python.exe exists

**Comment by Mammutor** [comment]
I also searched manually bit i didnt found python.exe. So I looked up where my python.exe is. and its in C:\Users\Mammu\AppData\Local\Programs\Python\Python38-32.
![C038NFg](https://user-images.githubusercontent.com/44206081/82658363-23699100-9c27-11ea-80a5-cc6dbe58a8ac.png)
And in Python38-32 `ls` works well.
![tLapRWx](https://user-images.githubusercontent.com/44206081/82658468-4eec7b80-9c27-11ea-9e31-ae28fabefec7.png)

**Comment by phanirithvij** [comment]
Did you install 32 bit python version by mistake? Install the 64 bit version after checking that your system is 64 bit. [Google](https://www.google.com/search?q=check+if+64+bit&oq=check+if+64&aqs=chrome.2.69i57j0l3.6223j0j7&client=ms-android-oppo&sourceid=chrome-mobile&ie=UTF-8). If it is 32 bit which I don't it is, add that ...python38-32 thing to path.

Make sure you install this if your system is 64 bit
![IMG_20200522_230808](https://user-images.githubusercontent.com/29627898/82694609-2b561f00-9c81-11ea-8f4a-e438593bcd20.jpg)

**Comment by Mammutor** [comment]
Yeah it was an acciedent. I have now also a Python Launcher installed. And I added it to PATH. Now I get a different message
![rqhZnYR](https://user-images.githubusercontent.com/44206081/82700807-3902a900-9c6f-11ea-9616-0c6a39dc6751.png)
I also changed the api.sh back to normal like on the file on github and did the same
![a6U0iga](https://user-images.githubusercontent.com/44206081/82701182-f42b4200-9c6f-11ea-9495-6d3612fcdf3f.png)
